src/app/booking/OOSPlannerSection.js

```diff
@@ -453,9 +489,8 @@ function BookingsTable() {
                       <ArrowForwardIosIcon />
                     </IconButton>
                   </Box>
-                  <Button startIcon={<IosShareIcon />} variant="outlined" color="grey">
-                    Share Calendar
-                  </Button>
+                  <ShareCalendar childrenData={children} showSharePopup={showSharePopup} />
+                  <ShareCalendarPopup open={shareCalendarPopupOpen} onClose={() => setShareCalendarPopupOpen(false)} childId={shareCalendarChildId} />
```

Comment:
Why do not mount it in filled booking? Ypu have to pass props all the way through.

Reply:
!